controllers/common_controller/include/ctrl.hpp

```diff
@@ -60,6 +60,12 @@
 /* Project includes */
 #include "odometry.hpp"
 
+#ifdef MODULE_UARTPB
+#include "uartpb/UartProtobuf.hpp"
+
+inline constexpr cogip::uartpb::uuid_t pose_reached_uuid = 2736246403;  ///< pose reached uuid for uartpb
+#endif
```

Comment:
Is it an UUID per message or per module ?

Reply:
Per message, and a module can handle multiple messages.

Reply:
:+1: